LEGALNOTICE.md

```diff
@@ -0,0 +1,69 @@
+COPYRIGHT
+---------
+
+OWASP Zed Attack Proxy (ZAP)
+
+The software package is:
+
+    Copyright (C) 2010  Simon Bennetts
```

Comment:
Could this be changed to something like "Copyright (C) 2010-2015 The ZAP Development Team" ?
I realize that the "ZAP Development Team" might be less well defined, but I dont think I should be taking credit for all the work other people contribute :)